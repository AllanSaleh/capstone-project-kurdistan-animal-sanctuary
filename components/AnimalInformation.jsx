import React from "react";
import PropTypes from "prop-types";

const AnimalInformation = ({ about, whatILike }) => (
  <div>
    <div>
      <h1 className="font-bold text-primary text-3xl pb-8">About me</h1>
      <p className="text-lg text-primary">{about}</p>
    </div>
    <div className="py-16">
      <h1 className="font-bold text-primary text-3xl pb-8">
        What I Like to do
      </h1>
      <p className="text-lg text-primary ">{whatILike}</p>
    </div>
  </div>
);
AnimalInformation.propTypes = {
  about: PropTypes.string.isRequired,
  whatILike: PropTypes.string.isRequired,
};
export default AnimalInformation;
