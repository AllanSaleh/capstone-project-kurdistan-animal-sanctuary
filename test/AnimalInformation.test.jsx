import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import AnimalInformation from "../components/AnimalInformation";

test("proparly renders the AnimalInformation component", () => {
  const tree = renderer.create(<AnimalInformation />).toJSON();
  expect(tree).toMatchSnapshot();
});
