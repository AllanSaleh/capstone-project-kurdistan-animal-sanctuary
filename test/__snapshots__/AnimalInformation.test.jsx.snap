// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the AnimalInformation component 1`] = `
<div>
  <div>
    <h1
      className="font-bold text-primary text-3xl pb-8"
    >
      animalProfile.aboutMe
    </h1>
    <p
      className="text-sm md:text-lg text-primary"
    />
  </div>
  <div
    className="py-16"
  >
    <h1
      className="font-bold text-primary text-3xl pb-8"
    >
      animalProfile.whatIlike
    </h1>
    <p
      className="text-sm md:text-lg text-primary "
    />
  </div>
</div>
`;
